admin-ui/src/app/shared/dialog/dialog.component.html

```diff
@@ -238,13 +238,15 @@ <h2 class="filter-heading">{{ "filters.titleTxt" | translate }}</h2>
 
 <!-- Message PopUp -->
 <div class="pop-up" *ngIf="input.case === 'MESSAGE'">
-  <img
-    src="assets/images/icon_Suceess right.png"
-    alt="Success Icon"
-    height="45px"
-    width="45px"
-    class="success-img"
-   *ngIf="input.title === 'Success' || input.title === 'نجاح' || input.title === 'Succès'|| input.title === 'सफलता' || input.title === 'ಯಶಸ್ಸು' || input.title === 'வெற்றி'"   />
+  <div class="pop-up-header">
+    <img
+      src="assets/images/icon_Suceess right.png"
+      alt="Success Icon"
+      height="45px"
+      width="45px"
+      class="success-img"
+     *ngIf="input.title === 'Success' || input.title === 'نجاح' || input.title === 'Succès'|| input.title === 'सफलता' || input.title === 'ಯಶಸ್ಸು' || input.title === 'வெற்றி'"   />
```

Comment:
Why are we considering "title" which is language-dependent, dont we have any field to give the code?

Reply:
Currently no as we hardcoded to display the icon based on title